src/components/VocabularyDrill/VocabularyDrill.tsx

```diff
@@ -240,12 +240,27 @@ export const VocabularyDrill: FC<VocabularyDrillProps> = ({
     onChangeLanguageMode(newLanguageMode);
   };
 
+  const handleShowResults = () => {
+    const newPage = page + 1;
```

Comment:
Not sure if that extra variable is really necessary (one could simply call `setPage(page + 1`), but it doesn't hurt much. Is there a reason why you introduced it?

Reply:
Oh, I see down below that it was there before. Feel free to simplify this ...